docs/DOCKER.md

```diff
@@ -1,11 +1,170 @@
-# How to run the service in docker
+# Running Connectors Service in Docker
 
-Make sure you have the latest `docker` command, then build the image:
+Running Connectors Service in Docker is really easy. All it requires is having Docker installed on a local system, and nothing more!
 
-    make docker-build
+This guide uses generally-available unix commands to demonstrate how to run the Connectors Service in Docker, for Windows you might have to run them in [Unix Subsystem](https://learn.microsoft.com/en-us/windows/wsl/about), rewrite the commands in PowerShell or do the steps in the guide using good old mouse and keyboard :)
 
+## Steps to properly run Connector Service in Docker
 
-To use it, tweak your [configuration file](./CONFIG.md) and run:
+1. Create a directory that will be mounted into the Docker image.
```

Comment:
```suggestion
### 1. Create a directory that will be mounted into the Docker image.
```

Consider using subheading for these main steps. Same comment for steps 2-4. :)

Reply:
Done! I also removed the numbers 1-4 to make it look better :)